tfgcv/testdata/full_compute_firewall.tf

```diff
@@ -0,0 +1,51 @@
+resource "google_compute_firewall" "full_list_default_1" {
+  name    = "test-firewall1"
+  network = "${google_compute_network.default.name}"
+
+  allow {
+    protocol = "icmp"
+  }
+  allow {
+    protocol = "tcp"
+    ports    = ["80", "8080", "1000-2000"]
+  }
+  description        = "test-description"
+  destination_ranges = ["test-destination_ranges1", "test-destination_ranges2"]
```

Comment:
I'm not sure about whether we should be using these hard-coded `test-*` values? In the future, there might be provider or conversion checks which ensure you're putting in valid values, and I'm not sure we actually benefit.

It seems like we'd be better off using an actual (but known) CIDR block here (or known zones, etc.).

Reply:
Agree. This is an initial attempt to build a sketelon of stuff that should be tested. I currently rely on `terraform plan` to complain if a value need to be changed. I also refer to the example in the documentation occasionally. 

I can fix the CIDR and zones.

Reply:
Got it, I can call out. the ones I think need changing

Reply:
updated with valid IP ranges.

Reply:
zone, region and location field are updated to use valid values.